traitsui/tests/test_actions.py

```diff
@@ -27,11 +27,16 @@
 from traitsui.item import Item
 from traitsui.view import View
 
-from traitsui.tests._tools import *
-from traitsui.tests._tools import _is_current_backend
+from traitsui.tests._tools import (
+    is_current_backend_null,
+    is_mac_os,
+    skip_if_not_qt4,
+    skip_if_not_wx,
+    store_exceptions_on_all_threads,
+)
 
-if _is_current_backend("null"):
-    raise nose.SkipTest("Not supported using the null backend")
+if is_current_backend_null():
+    raise unittest.SkipTest("Not supported using the null backend")
```

Comment:
This block is needed because `Action` class that is being instantiated just below is a toolkit object. Without this skip it would throw a `NotImplementedError`.

Also, I'm not sure if this PR would be the right place to address this anyways. The goal of this PR was to rewrite tests to use unittest without modifying their behaviour. So just like it wasn't possible to run this module directly with nose, it isn't possible to run it directly with unittest, even though the output is different.

Reply:
Ah I see.  I did not notice the class below can't even be instantiated without a backend.  While that's not a desirable interface, I agree that would be beyond the scope of this PR to change that.

I was hoping that I could do `python -m unittest traitsui.tests.test_actions` to see all the tests being skipped, rather than being greeted with an error. But that is very minor, and the outcome is the same: No tests cannot be run here.